### DISCUSSION TITLE
Will Red Hat integrate Tekton Results into their Tekton Pipelines Project and Support

### DISCUSSION THREAD

**Discussion by manuelwallrapp** [tektoncd/results#410] [Q&A] [answered]
Hi there

Are there any plans that Tekton-Results gets integrated into the Red Hat Tekton Project to also get paid support from Red Hat?
We are using Red Hat support for our Tekton issues, to also improve the product and our internal build experience. Would be great to have the same for Tekton-Results. 

Best regards

Manu

**Comment by khrm** [comment, marked as answer]
Yes, Results would be shipped in Openshift Pipelines operator 1.11

Expected results release 0.6.0

**Comment by adambkaplan** [comment]
@manuelwallrapp Please direct any further questions regarding Red Hat OpenShift Pipelines to https://issues.redhat.com/projects/SRVKP. You can follow our progress on integrating Results here: https://issues.redhat.com/ https://issues.redhat.com/browse/SRVKP-2721.

I want to be very clear here that this is an upstream project that is not tied to any specific product or company, though many contributors do work for companies that sell Tekton as a product or service. In that regard, @khrm I can't seem to find any issue or TEP adding Results to the Tekton Operator. Do you know how we should proceed here?

**Reply by vdemeester** [reply]
@adambkaplan results is already added to the operator (at least for the k8s part of it). We just need to make sure it's the case for the openshift part as well.

**Comment by adambkaplan** [comment]
Closing this discussion as answered.
